components/contribution-flow/query-parameters.js

```diff
@@ -26,10 +26,6 @@ const ContributionFlowUrlParametersConfig = {
    * @example 'month'
    */
   interval: { type: 'interval' },
-  /**
-   * A custom description
-   */
-  description: { type: 'string' },
```

Comment:
This one was unused and is not supported by GraphQLV2. Removed it from here and from the docs.